docs/source/index.rst

```diff
@@ -1,21 +1,43 @@
 Gene Normalizer |version|
 =========================
 
+.. image:: https://img.shields.io/pypi/pyversions/gene-normalizer?color=gr
+   :alt: PyPI - Python Version
+
+.. image:: https://github.com/cancervariants/gene-normalization/actions/workflows/github-actions.yml/badge.svg
+     :alt: tests status
+     :target: https://github.com/cancervariants/gene-normalization/actions/workflows/github-actions.yml
+
 .. image:: https://zenodo.org/badge/309797998.svg
      :alt: citation
      :target: https://zenodo.org/badge/latestdoi/309797998
 
-The Gene Normalizer provides tools for resolving ambiguous gene references to consistently-structured, normalized terms. For gene concepts extracted from `NCBI Gene <https://www.ncbi.nlm.nih.gov/gene/>`_, `Ensembl <https://useast.ensembl.org/index.html>`_, and `HGNC <https://www.genenames.org/>`_, it designates a `CURIE <https://en.wikipedia.org/wiki/CURIE>`_, and provides additional metadata like current and previously-used symbols, aliases, and database cross-references and associations. See the `public instance of the service <https://normalize.cancervariants.org/gene>`_ for a demonstration.
+The Gene Normalizer provides tools for resolving ambiguous human gene references to consistently-structured, normalized terms. For gene concepts extracted from `NCBI Gene <https://www.ncbi.nlm.nih.gov/gene/>`_, `Ensembl <https://useast.ensembl.org/index.html>`_, and `HGNC <https://www.genenames.org/>`_, it designates a `CURIE <https://en.wikipedia.org/wiki/CURIE>`_, and provides additional metadata like current and previously-used symbols, aliases, and database cross-references and associations.
```

Comment:
I know you already have a lot of examples of metadata, but might be good to highlight the locations as well?